Tasks/KubernetesManifestV0/src/utils/KubernetesObjectUtility.ts

```diff
@@ -205,7 +205,15 @@ export function getResources(filePaths: string[], filterResourceTypes: string[])
         const fileContents = fs.readFileSync(filePath);
         yaml.safeLoadAll(fileContents, function (inputObject) {
             const inputObjectKind = inputObject ? inputObject.kind : '';
+            let inputObjectStrategyType = '';
+            if(inputObject && inputObject.spec && inputObject.spec.updateStrategy){
+                inputObjectStrategyType = inputObject.spec.updateStrategy.type;
+            }
             if (filterResourceTypes.filter(type => isEqual(inputObjectKind, type, StringComparer.OrdinalIgnoreCase)).length > 0) {
+                if (checkStrategy && workloadTypesWithRolloutStatus.indexOf(inputObjectKind.toLowerCase()) >= 0 && !(inputObjectStrategyType === '' || isEqual(inputObjectStrategyType, "RollingUpdate", StringComparer.OrdinalIgnoreCase))) {
```

Comment:
Can you make this condition a little readable?

Reply:
As per chat I will make nested if conditions to make it readable.